core/unit_test/default/TestDefaultDeviceTypeInit_1.cpp

```diff
@@ -1,2 +0,0 @@
-#define KOKKOS_DEFAULTDEVICETYPE_INIT_TEST_01
-#include <TestDefaultDeviceTypeInit.hpp>
```

Comment:
Why is this include getting deleted from all `TestDefaultDeviceTypeInit_N.cpp`?

Reply:
These tests are deprecated, see https://github.com/kokkos/kokkos/blob/b2371de0d71192e02d3cdb300ba5cf937d79f4ee/core/unit_test/CMakeLists.txt#L884-L891.